system/BaseModel.php

```diff
@@ -382,7 +383,7 @@ abstract protected function doFindColumn(string $columnName);
      *
      * @return array
      */
-    abstract protected function doFindAll(int $limit = 0, int $offset = 0);
+    abstract protected function doFindAll(?int $limit = null, int $offset = 0);
```

Comment:
nullable flag seems not needed, just default null should be ok, see https://3v4l.org/4NMIj that also make suggest to apply null only when they know what they are doing

Reply:
The rule https://cs.symfony.com/doc/rules/function_notation/nullable_type_declaration_for_default_null_value.html adds `?`.

Reply:
Ok 👍